src/components/Item/AppItem/getIframeSrc.js

```diff
@@ -11,5 +11,12 @@ export const getIframeSrc = (appDetails, item, itemFilters) => {
         iframeSrc += `&userOrgUnit=${ouIds.join(',')}`
     }
 
+    // Add period (pe) to iframeSrc
+    if (itemFilters[FILTER_PE] && itemFilters[FILTER_PE].length) {
```

Comment:
I know that this is the same logic as the expression above, but [Sonarqube](https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=3183&id=dhis2_dashboard-app&open=AZRoa7daNcCCOAJdJDaq) recommends using an optional chaining expression here:
```suggestion
    if (itemFilters[FILTER_PE]?.length) {
```